Flow.Launcher/Converters/TextConverter.cs

```diff
@@ -10,23 +10,17 @@ public class TextConverter : IValueConverter
     {
         public object Convert(object value, Type targetType, object parameter, CultureInfo culture)
         {
-            var ID = value.ToString();
-            switch(ID)
+            var id = value?.ToString();
+            return id switch
```

Comment:
maybe let this expression returns the key and merge the `InternationalizationManager.Instance.GetTranslation(key)`